app/controllers/recommendations_controller.rb

```diff
@@ -22,8 +30,57 @@ def create
     end
   end
 
+  def review_form
```

Comment:
Could we arguably change `review_form` to `edit`, and `review` to `update`? 

If not I'm almost inclined to say we should have `reviews_controller` instead, with `new` and `create`. It would be really nice to only stick 100% to restful actions 🤔

Reply:
Yeah I think you are right.. not sure why i didn't frame it like this